examples/tictactoe/src/components/Connect.tsx

```diff
@@ -26,11 +26,11 @@ const styles = {
     justifyContent: 'center',
     alignItems: 'center',
     height: '100vh',
-    backgroundColor: '#f5f5f5',
+    backgroundColor: '#F2F3F8',
   },
   content: {
     display: 'flex',
-    flexDirection: 'column' as 'column',
+    flexDirection: 'column' as const,
```

Comment:
Might be worth to either add a type to the `const styles` or just do the `as const` for the whole `const styles`

Reply:
cool can actually remove the as `const`